ckanext/protected_resources/plugin.py

```diff
@@ -116,15 +82,22 @@ def protected_resource_lock(context, data_dict):
 
     resource = p.toolkit.get_action('resource_show')(context, {'id': data_dict['resource_id']})
 
-    resource['is_protected'] = True
```

Comment:
5/10
Line 67
Needs to be updated to reflect the new implementation:
```
This will update the resource extras field with an is_protected: True value
```

Reply:
fixed